Assets/Scripts/Framework/Animation/AnimationController.cs

```diff
@@ -0,0 +1,39 @@
+using System.Linq;
+using UnityEngine;
+
+namespace Framework.Animation
+{
+    public sealed class AnimationController : MonoBehaviour
+    {
+        private Animator _animator;
+
+        private const string INVALID_ANIMATION = "Animation name does not exist in the Animator controller: ";
+        private const string NO_ANIMATOR_ERROR = "Animator component is not assigned.";
```

Comment:
```suggestion
        private const string INVALID_ANIMATION = "Animation name does not exist in the Animator controller: ";
        private const string NO_ANIMATOR_ERROR = "Animator component is not assigned.";
        
        private Animator _animator;
```
[Consts moeten boven aan.](https://github.com/Team-Swamp/IceBites/wiki/Code-convention#classes)